src/Build/BackEnd/Components/RequestBuilder/IntrinsicTasks/ItemGroupLoggingHelper.cs

```diff
@@ -114,7 +114,7 @@ internal static string GetStringFromParameterValue(object parameterValue)
 
                 var customMetadata = item.CloneCustomMetadata();
```

Comment:
It would be nice to avoid this, too, when logItemMetadata is false.

Reply:
Once https://github.com/microsoft/msbuild/pull/5224 is merged, I'll have to fix merge conflicts and will fix it then :)

Reply:
Fixed.